setup.py

```diff
@@ -11,14 +10,27 @@ def read(*names, **kwargs):
         return fh.read()
 
 
+def get_version():
+    version_file = Path("src/laser_core/__init__.py")
+    version_match = re.search(r'^__version__ = ["\']([^"\']+)["\']', version_file.read_text(encoding="utf8"), re.M)
+    if version_match:
+        return version_match.group(1)
+    raise RuntimeError("Unable to find version string.")
+
+
 setup(
+    name="laser_core",  # Specify the name of your package
+    version=get_version(),  # Fetch version from __init__.py
+    description="A description of the laser_core package",  # Add a description for the package
```

Comment:
All of this is redundant with pyproject.toml and should not be here. The `long_description_content_type` can stay.

Reply:
> All of this is redundant with pyproject.toml and should not be here. The `long_description_content_type` can stay.

Ah, ok. I guess I was building with setuptools and getting wheels without versions or names, otherwise I wouldn't have added it. I'll see about building with toml.